src/Fixer/ConstantNotation/NativeConstantInvocationFixer.php

```diff
@@ -0,0 +1,171 @@
+
+            $tokenContent = $token->getContent();
+
+            // test if we are at a constant call
+            if (!$token->isGivenKind(T_STRING)) {
+                continue;
+            }
+
+            $next = $tokens->getNextMeaningfulToken($index);
+            if ($tokens[$next]->equals('(')) {
+                continue;
+            }
+
+            $constantNamePrefix = $tokens->getPrevMeaningfulToken($index);
+            if ($tokens[$constantNamePrefix]->isGivenKind([CT::T_USE_TRAIT, T_CLASS, T_CONST, T_DOUBLE_COLON, T_EXTENDS, T_FUNCTION, T_IMPLEMENTS, T_INTERFACE, T_NAMESPACE, T_NEW, T_NS_SEPARATOR, T_OBJECT_OPERATOR, T_TRAIT, T_USE])) {
+                continue;
+            }
+
+            if (!\in_array($tokenContent, $constantNames, true)) {
+                continue;
+            }
+
+            $indexes[] = $index;
+        }
+
+        $indexes = \array_reverse($indexes);
+        foreach ($indexes as $index) {
+            $tokens->insertAt($index, new Token([T_NS_SEPARATOR, '\\']));
+        }
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function createConfigurationDefinition()
+    {
+        $constantChecker = static function ($value) {
+            foreach ($value as $constantName) {
+                if (!\is_string($constantName) || '' === \trim($constantName) || \trim($constantName) !== $constantName) {
+                    throw new InvalidOptionsException(\sprintf(
+                        'Each element must be a non-empty, trimmed string, got "%s" instead.',
+                        \is_object($constantName) ? \get_class($constantName) : \gettype($constantName)
+                    ));
+                }
+            }
+
+            return true;
+        };
+
+        return new FixerConfigurationResolver([
+            (new FixerOptionBuilder('fix_built_in', 'Whether to fix constants returned by `get_defined_constants`.'))
+                ->setAllowedTypes(['boolean'])
+                ->setDefault(true)
+                ->getOption(),
+            (new FixerOptionBuilder('include', 'List of additional constants to fix.'))
+                ->setAllowedTypes(['array'])
+                ->setAllowedValues([$constantChecker])
+                ->setDefault([])
+                ->getOption(),
+            (new FixerOptionBuilder('exclude', 'List of constants to ignore.'))
+                ->setAllowedTypes(['array'])
+                ->setAllowedValues([$constantChecker])
+                ->setDefault(['NULL', 'FALSE', 'TRUE'])
+                ->getOption(),
+        ]);
+    }
+
+    /**
+     * @return string[]
+     */
+    private function getConstantNames()
+    {
+        $definedConstants = $this->configuration['include'];
+        if (true === $this->configuration['fix_built_in']) {
+            $definedConstants = array_merge($definedConstants, \array_keys(\get_defined_constants()));
+        }
+
+        return \array_diff(
+            $definedConstants,
```

Comment:
as we call `array_unique` on `exclude`, we could call `array_unique` on `$definedConstants` as well (as I could include sth twice (same as I exclude sth twice and you handle it), and even include sth that is included by `fix_build_in`)